backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -513,17 +344,10 @@ private async void OnPressureUpdate(object? sender, MqttReceivedArgs mqttArgs)
                 await timeseriesService.Create(
                     new RobotPressureTimeseries
                     {
-                        MissionId = robot.CurrentMissionId,
-                        Pressure = pressureStatus.PressureLevel,
-                        RobotId = robot.Id,
-                        Time = DateTimeOffset.UtcNow
+                        MissionId = robot.CurrentMissionId, Pressure = pressureStatus.PressureLevel, RobotId = robot.Id, Time = DateTimeOffset.UtcNow
```

Comment:
This is again personal preference but I do find splitting up these assignments across several lines to be more readable.

Reply:
At least when there’s more than a couple variables.

Reply:
This one is the one setting I can't find in Rider as I agree with you. Dotnet format also requires it so it will be fixed.